### ISSUE TITLE
The Journey Down Chapter Two/Three segfault on launch

### ISSUE THREAD

**Issue by jurf** [flathub/com.valvesoftware.Steam#347]
**Game information**
[The Journey Down: Chapter Three](https://store.steampowered.com/app/579760/The_Journey_Down_Chapter_Three/)

**Distribution name and version where applicable**
Silverblue 30.

**Flatpak info**

```
$ flatpak --version
Flatpak 1.2.4
$ flatpak --gl-drivers
flatpak --gl-drivers
default
host
```

**Problem description**
Simple segfault on launch:
```
$ ./JourneyDown3 
Data directory: /home/jurf/.config/JourneyDownThree Archive path: /var/home/data/games/steamapps/common/The Journey Down Chapter Three/jd3.gob

Initializing Theoraplayer Video Playback Library (2.0)
  - libtheora version: Xiph.Org libtheora 1.2.0alpha 20100924 (Ptalarbvorm)
  - libvorbis version: Xiph.Org libVorbis 1.3.4
------------------------------------
Changing number of worker threads to: 1
Starting The Journey Down: Chapter 3.09
Segmentation fault (core dumped)
```

Coredump: [core.JourneyDown3.1000.243015d27cac419c9297e769bb42f312.17480.1554733905000000.zip](https://github.com/flathub/com.valvesoftware.Steam/files/3056178/core.JourneyDown3.1000.243015d27cac419c9297e769bb42f312.17480.1554733905000000.zip)

**Does this issue reproduce with native Steam**
Have not tested, though probably not, because I played Chapter Two on native Steam and it worked fine, but doesn’t work in Flatpak.

**Comment by mdosch** [comment]
Afaik, Fedora uses wayland. When using sway it also segfaulted for me but when using i3 all was good. I guess you should also try playing it from an Xorg WM/DE. See https://steamcommunity.com/linkfilter/?url=https://github.com/flathub/com.valvesoftware.Steam/issues/347

**Comment by jurf** [comment]
Yeah, this is right. I wonder if this is a fault in XWayland…

**Comment by AsciiWolf** [comment]
@mdosch The url does not seem to be right. :-)

**Comment by mdosch** [comment]
You're right, that's the correct link: https://steamcommunity.com/app/579760/discussions/0/1814296273140602243/#c3022387599802270491

**Comment by nanonyme** [comment]
Not yet giving up on this. I want to see with 19.08 runtime if we can support this better than official Steam.

**Comment by jurf** [comment]
Hm, though is this something we can fix? I mean,

> Yeah sorry, Wayland is not supported I'm afraid. 

doesn’t sound too optimistic, especially when you look at the logs.

**Comment by nanonyme** [comment]
I guess you're right.
